csrankings-a.csv

```diff
@@ -206,6 +206,7 @@ Adam Herout,Brno University of Technology,https://www.fit.vut.cz/person/herout,q
 Adam J. Aviv,George Washington University,https://adamaviv.com,ExgxufEAAAAJ
 Adam J. Lee,University of Pittsburgh,http://people.cs.pitt.edu/~adamlee,1IoFK0oAAAAJ
 Adam Jatowt,University of Innsbruck,https://adammo12.github.io/adamjatowt,l2vn9GoAAAAJ
+Adam Klivans,University of Texas at Austin,https://www.cs.utexas.edu/~klivans,UD87zMYAAAAJ

```

Comment:
Adam is already in the database and this is not his DBLP name (https://dblp.org/pid/k/AdamRKlivans.html).